apps/backoffice-v2/src/common/env/schema.ts

```diff
@@ -17,4 +17,5 @@ export const EnvSchema = z.object({
     .or(z.literal(false))
     .catch(undefined),
   VITE_IMAGE_LOGO_URL: z.string().optional(),
+  WEBHOOK_URL: z.string().url().default('ws://localhost:3500'),
```

Comment:
there is an `.env` that I missed which make the app get `❌ Invalid environment variables`

Reply:
Hey @TzlilSwimmer123, can we resolve the comments and fix the merge conflicts?

Reply:
on it, left a question